Paintroid/src/main/java/org/catrobat/paintroid/MainActivity.java

```diff
@@ -296,14 +298,66 @@ protected void onDestroy() {
 		PaintroidApplication.savedPictureUri = null;
 		PaintroidApplication.saveCopy = false;
 
-		PaintroidApplication.commandManager.storeCommandLists();
 		PaintroidApplication.commandManager.setInitialized(false);
+		PaintroidApplication.commandManager.resetAndClear(false);
+
+		PaintroidApplication.colorPickerInitialColor = Color.BLACK;
 
 		IndeterminateProgressDialog.getInstance().dismiss();
 		ColorPickerDialog.getInstance().dismiss();
 		super.onDestroy();
 	}
 
+	@Override
+	public void onConfigurationChanged(Configuration newConfig) {
+		super.onConfigurationChanged(newConfig);
+
+		if (newConfig.orientation == 2)
+			getWindow().addFlags(WindowManager.LayoutParams.SCREEN_ORIENTATION_CHANGED);
+		else
+			getWindow().clearFlags(WindowManager.LayoutParams.SCREEN_ORIENTATION_CHANGED);
```

Comment:
remove adding and clearing SCREEN_ORIENTATION_CHANGED flags, status bar issues should be fixed with #406 PAINT-218